plugins/watcher/go.mod

```diff
@@ -0,0 +1,80 @@
+module github.com/fairwindsops/insights-plugins/plugins/watcher
+
+go 1.24.6
+
+require (
+	github.com/aws/aws-sdk-go-v2 v1.32.2
+	github.com/aws/aws-sdk-go-v2/config v1.28.0
+	github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs v1.36.0
+	github.com/sirupsen/logrus v1.9.3
```

Comment:
[opt] since this will be a data intensive app, maybe change from `logrus` to `slog`

Reply:
done